### ISSUE TITLE
about how to train ssd

### ISSUE THREAD

**Issue by ZhouYuSong** [weiliu89/caffe#32]
I trained the ssd300,but the result only reach 55%~,can you explain the details about how you train the final model?

**Comment by weiliu89** [comment]
What have you changed? Can you provide more details?

**Comment by ZhouYuSong** [comment]
only the batchsize,change it to 2.......

**Comment by weiliu89** [comment]
Do you change [both lines](https://github.com/weiliu89/caffe/blob/ssd/examples/ssd/ssd_pascal.py#L290-L291) to 2? If so, you would like to increase your training iterations proportionally. Besides, probably you should adjust your learning rate a bit as well.

**Comment by ZhouYuSong** [comment]
ok,thank you! I'll have a try.

**Comment by conner99** [comment]
I set a batch_size of 4

and add this to ssd_pascal.py, because the post in BVLC#430 (comment)

```
#if batch_size is divided by X then max_iter and stepsize should multiply by X
max_iter = 60000 * (32 / batch_size)
stepsize = 40000 * (32 / batch_size)
```

**Comment by weiliu89** [comment]
Actually, the accum_batch_size defined in [here](https://github.com/weiliu89/caffe/blob/ssd/examples/ssd/ssd_pascal.py#L290-L292) has solved your problem. You could change batch_size to 4 and keep accum_batch_size to 32.

**Comment by conner99** [comment]
Thank you for the clue and the great work!

**Comment by abhisheksgumadi** [comment]
Hi, what is the stepsize parameter during training?

**Comment by weiliu89** [comment]
It should be the same if you don't change accum_batch_size.

**Comment by abhisheksgumadi** [comment]
Thanks Wei, just to clarify did you mean it should have the same value as accum_batch_size?
